packages/compiler/src/renderers/components/ImageView/index.tsx

```diff
@@ -0,0 +1,119 @@
+/******************************************************************
+ * created by lqy at 06/16/2021
+ ******************************************************************/
+
+import React from 'react';
+import Modal from '@material-ui/core/Modal';
+import Fade from '@material-ui/core/Fade';
+import { makeStyles, createStyles } from '@material-ui/styles';
+
+export interface PreviewProps {
```

Comment:
The name of the interface should start with the name of the `Component`, the name here should be: `ImageViewProps`.
```suggestion
export interface ImageViewProps {
```